src/Algebra/Lattice/Dropped.hs

```diff
@@ -103,6 +104,10 @@ instance Finite a => Finite (Dropped a) where
     universeF = Top : map Drop universeF
     cardinality = fmap succ (retag (cardinality :: Tagged a Natural))
 
+nothingOnTopOfJust :: Maybe a -> Dropped a
+nothingOnTopOfJust Nothing = Top
+nothingOnTopOfJust (Just a) = Drop a
```

Comment:
this is just `maybe Top Drop`, and the other one is `maybe Bottom Lift`, so doesn't pass [Fairbairn threshold](https://wiki.haskell.org/Fairbairn_threshold)

Reply:
Makes sense, didn't know it's named anti-pattern, thanks. Function removed.